packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -35,6 +36,12 @@ class Popover extends React.Component {
      */
     classes: PropTypes.array,
 
+    /**
+     * Whether the popover should be closed when its closest scrollable parent is scrolled.
+     * [Default] true
+     */
+    closeWhenParentScrolled: PropTypes.bool,
```

Comment:
This could just be closeOnScrolled or something similar. The idea is that it gets closed when it moves... original just seems a little wordy to me. Not a huge deal though